code/game/objects/structures/crates_lockers/closets/secure/hydroponics.dm

```diff
@@ -11,3 +11,90 @@
 	new /obj/item/cultivator(src)
 	new /obj/item/hatchet(src)
 	new /obj/item/secateurs(src)
+
+//Plocker
+/obj/structure/closet/secure_closet/plocker
```

Comment:
making this a subtype of /obj/structure/closet/decay could remove the need for some redundant code